src/views/Orders.vue

```diff
@@ -277,7 +277,40 @@ export default {
     IonTitle,
     IonToolbar
   },
+  computed: {
+    ...mapGetters({
+      orders: 'order/getList',
+      getProduct: 'product/getProduct',
+      currentFacilityId: 'user/getCurrentFacility'
+    })
+  },
+  methods: {
+    async getOrders(vSize?: any, vIndex?: any){
```

Comment:
Please confirm if you have used <script lang="ts"> instead of <script> as you are also using typescript here.